demos/_includes/DemoInvoice.php

```diff
@@ -25,6 +25,9 @@ protected function init(): void
                 function ($v) {
                     return ($v instanceof \DateTime) ? date_format($v, $this->dateFormat) : $v;
                 },
+                function ($v) {
```

Comment:
I wonder, what is default, when the 2nd typecast closure is not specified.

Reply:
Not sure what you mean by default but when the closure is not specify here is the Exceptions thrown.

<img width="922" alt="Screen Shot 2021-01-27 at 8 28 12 AM" src="https://user-images.githubusercontent.com/2204478/105997662-dbcebb00-6079-11eb-980d-e5d220cb664f.png">
<img width="919" alt="Screen Shot 2021-01-27 at 8 28 25 AM" src="https://user-images.githubusercontent.com/2204478/105997678-e12c0580-6079-11eb-9ef6-03f7ae85603f.png">

Reply:
I mean what typecast is used when it is not specified.

Reply:
This is used with Persistence\Array_, not sure about this but to my understanding, Array_ persistence does not provide default typecast..

Reply:
the exceptions seems to provide "default typecast"... Otherwise it will not warn and dummy closure will be not needed

Reply:
so I think the issue is somewhere else

Reply:
Looks like a data  Persistence\Array issue. Will merge has is for now so the demo does not crash.